content/60.methods.md

```diff
@@ -106,8 +106,11 @@ Due to the needs of this project, project contributors also implemented new feat
 Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations.
 These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].
 
-Additionally, the fast-moving nature of the infodemic has led to a number of retractions and corrections of COVID-19 literature.
-<!--To Do: Describe integration with scite -- talk to Vince-->  
+A new plugin was also added to Manubot to support "smart citations".
+The plugin uses the [Scite](https://scite.ai/) service to display a badge below each citation with a set of icons and numbers indicating how many times that source has been mentioned, supported, or disputed.
+The badge also indicates important editorial notices, such as retractions and corrections.
+This allows readers to roughly evaluate the reliability of cited sources at a glance.
+In this project, where we are disseminating rapidly evolving information of great consequence from over a thousand different sources, we found this feature invaluable.
```

Comment:
```suggestion
In this project, where we were disseminating rapidly evolving information of great consequence from over a thousand different sources, this feature offered the invaluable function of streamlining the identification of references that needed to be double checked as time passed from when they were added.
```
I'm not quite sure if this does it, but trying to add in a mention of the temporal aspect of this challenge!